src/themis/secure_message.c

```diff
@@ -52,10 +52,22 @@ themis_status_t themis_gen_rsa_key_pair(uint8_t* private_key,
 				    uint8_t* public_key,
 				    size_t* public_key_length){
   soter_rsa_key_pair_gen_t* key_pair_ctx=soter_rsa_key_pair_gen_create(THEMIS_RSA_KEY_LENGTH);
-  themis_status_t res=soter_rsa_key_pair_gen_export_key(key_pair_ctx, private_key, private_key_length, true);
-  themis_status_t res1=soter_rsa_key_pair_gen_export_key(key_pair_ctx, public_key, public_key_length, false);
+  THEMIS_CHECK(key_pair_ctx!=NULL);
+  soter_status_t res=soter_rsa_key_pair_gen_export_key(key_pair_ctx, private_key, private_key_length, true);
+  if(res!=THEMIS_SUCCESS && res != THEMIS_BUFFER_TOO_SMALL){
```

Comment:
`res != THEMIS_BUFFER_TOO_SMALL` -> `res!=THEMIS_BUFFER_TOO_SMALL` ?)
but I like with space between values because then it doesn't look like one looong variable (which has some comparison signs inside)

Reply:
Me too, but I wanted to make it consistent across the file (which I obviously missed here).